src/Audio/Listener.cs

```diff
@@ -69,28 +69,28 @@ public static Vector3f UpVector
         }
         
         #region Imports
-        [DllImport("csfml-audio-2", CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
```

Comment:
Found a comment from a CoreCLR dev: https://github.com/dotnet/coreclr/issues/1605
It's essentially a "built-in" feature for .NET Core.

So yes, not needed in .NET Core, but would be needed for normal .NET.